csi.proto

```diff
@@ -947,6 +947,39 @@ message GetCapacityResponse {
   // storage. This field is REQUIRED.
   // The value of this field MUST NOT be negative.
   int64 available_capacity = 1;
```

Comment:
Now that we have added the new capacity fields for max/min volume size, can we also add comments to this existing field to explain how CO should be using this field?

Reply:
I think that is not clear yet. If you have a proposal, then please put it into a PR and we can discuss it.